index.html

```diff
@@ -107,20 +107,20 @@ <h1>About <br> MySelf</h1>
             <h3>LET'S  CONNECT</h3>
      </div>
           <div class="icon">
-            <img src="image/image 6.png" alt="twitter">
+            <img src="image/image6.png" alt="twitter">
             <img src="image/image7.png" alt="twitter">
             <img src="image/image8.png" alt="twitter">
             <img src="image/image9.png" alt="twitter">
             <img src="image/imge5.png" alt="twitter">
           </div>
```

Comment:
- [ ] Please icons should be inside a ul tag since it represent a list of elements
![image](https://user-images.githubusercontent.com/74744787/185768060-0fa9051c-c78a-4b99-a787-b95551f36d3a.png)